### PR TITLE
Improvements to App Platform overview page

### PR DESCRIPTION
**Pull request by rossf7**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/master/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/app-platform/overview/index.md [hunk 1]
```diff
@@ -30,11 +30,12 @@ The _Giant Swarm App Platform_ refers to a set of features and concepts that all
 you to browse, install and manage the configurations of apps (such as Prometheus)
 from a single place; the management cluster.
 
-Using this platform we will provide a collection of curated _Apps_. These _Apps_ are grouped into _App Catalogs_, which are browsable through our web interface.
+Using this platform we provide a collection of curated _Apps_. These _Apps_ are grouped into _App Catalogs_, which are browsable through our web interface.
```
Comment (rossf7) [inline src/content/app-platform/overview/index.md:33]: I think this should be present tense.

File: src/content/app-platform/overview/index.md [hunk 2]
```diff
@@ -30,11 +30,12 @@ The _Giant Swarm App Platform_ refers to a set of features and concepts that all
...
 
 In short: the _Giant Swarm App Platform_ refers to the whole feature, and an _App Catalog_ is a collection of _Apps_.
 
-We provide two _App Catalogs_. You will be able to set up your own additional catalog(s) to provide for any needs you have at the enterprise level.
+We provide two _App Catalogs_, the Giant Swarm Catalog and the Giant Swarm Playground. You are able to set up your own additional catalog(s)
```
Comment (rossf7) [inline src/content/app-platform/overview/index.md:37]: Not naming the catalogs reads strangely to me. Especially as we use the names later.

Some customers have created their own catalogs so I think this should also be present tense.

**Comment by rossf7** [comment]
Some updates to the overview page.

app-platform-overview

@oponder Do you know where the diagram is for this? It would be great to update the naming and remove tiller.

File: src/content/app-platform/overview/index.md (hunk 1 above)
Comment (tomahawk28) [inline src/content/app-platform/overview/index.md:33]:
```suggestion
Using this platform, we are providing a collection of curated _Apps_. These _Apps_ are grouped into _App Catalogs_, which are browsable through our web interface.
```
WDYT present continuous? 😄

**Review by tomahawk28 — APPROVED** [review]

File: src/content/app-platform/overview/index.md (hunk 1 above)
Comment (rossf7) [reply src/content/app-platform/overview/index.md:33]: Love it. <3 Day 2 app ops.

**Review by marians — APPROVED** [review]

**Pull request merged**